app/api/custom/orders.py

```diff
@@ -145,3 +145,35 @@ def create_order():
     order.populate_and_save()
 
     return OrderSchema().dump(order)
+
+
+@order_blueprint.route('/attendees/<int:attendee_id>.pdf')
+@jwt_required
+def ticket_attendee_pdf(attendee_id):
+    if current_user:
+        try:
+            ticket_holder = TicketHolder.query.get(attendee_id)
+            if (
+                current_user.is_staff
+                or ticket_holder.order.user_id == current_user.id
+                or ticket_holder.user == current_user
+                or has_access(
+                    'is_coorganizer_or_user_itself',
+                    event_id=ticket_holder.event_id,
+                    user_id=ticket_holder.user.id,
+                )
+            ):
+
+                file_path = ticket_holder.pdf_url
+                try:
+                    return send_from_directory('../', file_path, as_attachment=True)
+                except FileNotFoundError:
+                    return NotFoundError({'source': ''}, 'File Not found')
```

Comment:
This will fail if the file doesn't exist

Reply:
What shall be the behaviour?